### PR TITLE
Auth page

### PR DESCRIPTION
**Pull request by iRohitGaur**
login
signup

Files to check:
- [Check these files](https://github.com/iRohitGaur/StarCart/pull/10/files?file-filters%5B%5D=.jsx&show-viewed-files=true)

Files to ignore:
- all css files

### CODE REVIEW

**Review by b0llu — APPROVED** [review]
Great code and clean also. Easy to read. Nothing to suggest. Suggested one change

**Review by srishtigautam21 — APPROVED** [review]
I get to learn new things by reviewing your PR 
The code is well understandable
Good to go

File: src/pages/auth/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        >
+          use guest login
+        </button>
+        <div className="flex_row flex_gap1">
+          <input type="checkbox" value="" />
+          <span className="login_checkbox_text">Remember me</span>
+        </div>
+      </div>
+
+      <button className="sui_btn ep_mt2">Login</button>
```
Comment (b0llu) [inline src/pages/auth/Login.jsx]: Suggestion - bring the validation logic from input.jsx to this so that it can help in checking at both ends

**Comment by santhoshpemmaka** [comment]
Code looks well understandable and has good structure. No suggestions from my side. Great Work 💯 . Good to Merge 👍

**Review by santhoshpemmaka — APPROVED** [review]

File: src/pages/auth/Login.jsx (hunk 1 above)
Comment (iRohitGaur) [reply src/pages/auth/Login.jsx]: Thanks for pointing it out. Done!

**Pull request merged**